avocado/plugins/docker.py

```diff
@@ -186,10 +185,6 @@ def configure(self, parser):
         cmd_parser.add_argument("--docker-options", default="",
                                 help="Extra options for docker run cmd."
                                 " (see: man docker-run)", metavar="OPT")
-
-        cmd_parser.add_argument("--docker-no-copy", action="store_true",
-                                help="Assume tests are already in the "
-                                "container")
```

Comment:
The `plugins.docker`'s `DockerRemoter` can (and should) be significantly simplified with the removal of docker_no_copy.